upup/pkg/fi/cloudup/vspheretasks/attachiso.go

```diff
@@ -57,6 +94,73 @@ func (_ *AttachISO) CheckChanges(a, e, changes *AttachISO) error {
 }
 
 func (_ *AttachISO) RenderVC(t *vsphere.VSphereAPITarget, a, e, changes *AttachISO) error {
```

Comment:
Task framework calls Render* method. I'll rename RenderVC method to RenderVSphere to keep it consistent with "VSphere" we have been using everywhere else.